src/shared/pager.c

```diff
@@ -161,8 +161,8 @@ void pager_close(void) {
                 return;
 
         /* Inform pager that we are done */
-        stdout = safe_fclose(stdout);
-        stderr = safe_fclose(stderr);
+        safe_fclose(stdout);
+        safe_fclose(stderr);
```

Comment:
Trying to write to stdout/stderr after they are freed would produce a segfault in both cases. Perhaps doing freopen("/dev/null","w",stdout) might be a better idea.